WordPress/Classes/ViewRelated/Blog/Add Site/AddSiteAlertFactory.swift

```diff
@@ -0,0 +1,51 @@
+import Foundation
+
+/// This class takes care of constructing our "Add Site" action sheets.  It does not handle any presentation logic and does
+/// not know any external data sources - all of the data is received as parameters.
+@objc
+class AddSiteAlertFactory: NSObject {
+
+    @objc
+    func make(
```

Comment:
maybe we could use a more specific name here (e.g. makeAddSiteAlert or something like that)?

Reply:
Changed.